### DISCUSSION TITLE
🐸 Discord Server... Join in !!

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#2128] [Announcements]
## We created a **Discord** server, and 🫵 are [invited](https://discord.gg/5eXr5seRrv)!

Click here to join 👉  https://discord.gg/5eXr5seRrv

### Why discord?

We think Discord creates a better community feeling and visibility among people and channels. 

### What happens to the previous channels?

ATM, we plan to keep all the previous channels (Gitter, Matrix, ElementIO) active until we find a better solution to migrate, bridge, etc. 

✉️ Please let us know your feedback!!

**Comment by riyaelizabeth1318** [comment]
I want to convert one voice to another input voice

**Reply by riyaelizabeth1318** [reply]
how can i do that
